packages/modules/promotion/src/utils/compute-actions/usage.ts

```diff
@@ -34,3 +58,16 @@ export function computeActionForBudgetExceeded(
     }
   }
 }
+
+export function getBudgetUsageContextFromComputeActionContext(
```

Comment:
thought: Using a function like this prevents users from specifying other attributes.

Can't we just pass these properties down as is? I.e. we pass `customer_email` and `customer_id` in the context.